src/App/Components/DisplayArticle/readArticle.js

```diff
@@ -77,44 +75,51 @@ export class ReadArticle extends Component {
 
   render() {
     const {
-      category, title, body, readTime, tags,
-      createdAt, coverImage,
-      firstName, lastName, profileImage, slug, isLoading, articles
+      category,
+      title,
+      body,
+      readTime,
+      tags,
+      createdAt,
+      coverImage,
+      firstName,
+      lastName,
+      profileImage,
+      slug,
+      isLoading,
+      articles
     } = this.state;
 
-    return isLoading ? <Loader /> : (
+    return isLoading ? (
+      <Loader />
+    ) : (
       <React.Fragment>
-        {
-          !slug ? <ArticleNotFound content="ARTICLE NOT FOUND" /> : (
-            <div>
-              <Navbar />
-              <BreadCrumb category={category} />
-              <div className="container article">
-                <div className="row">
-                  <ArticleBody
-                    title={title}
-                    body={body}
-                    readTime={readTime}
-                    tags={tags}
-                    createdAt={createdAt}
-                    coverImage={coverImage}
-                    firstName={firstName}
-                    lastName={lastName}
-                    profileImage={profileImage}
-                  />
-
-
-                  {/* <div className="col-xl-2 col-lg-2 col-md-12 col-sm-12 col-12">
-                    <Popular articles={articles} />
-                  </div> */}
-
-                  <BottomPopular articles={articles} />
-                </div>
+        {!slug ? (
+          <ArticleNotFound content="ARTICLE NOT FOUND" />
+        ) : (
+          <div>
+            <Navbar />
+            <BreadCrumb category={category} />
+            <div className="container article">
+              <div className="row">
+                <ArticleBody
+                  title={title}
+                  body={body}
+                  readTime={readTime}
+                  tags={tags}
+                  createdAt={createdAt}
+                  coverImage={coverImage}
+                  firstName={firstName}
+                  lastName={lastName}
+                  profileImage={profileImage}
+                />
+                <ArticleSocialSharing slug={slug} title={title} />
```

Comment:
social sharing is shorter and reasonable, don't you think @murediane

Reply:
alright .let me work on that